format/index.md

```diff
@@ -919,7 +919,8 @@ a properly generated and formatted *UUID* as described in RFC 4122
 > NOTE: In some use-cases, such as importing data from another source, it
 may be possible to use something other than a UUID that is still guaranteed
 to be globally unique. Do not use anything other than a UUID unless you are
-100% confident that the strategy you are using is indeed globally unique.
+100% confident that the strategy you are using indeed generates globally 
+unique indentifiers.
```

Comment:
@steveklabnik, sorry, not sure I understand... I thought IDs should be unique. How could uniqueness of stategy be important from the POV of JSON API?